controllers/state_of_the_world.go

```diff
@@ -63,15 +71,122 @@ func NewPolicyMachineryController(manager ctrlruntime.Manager, client *dynamic.D
 }
 
 func buildReconciler(client *dynamic.DynamicClient) controller.ReconcileFunc {
+	authorinoCrSubscription := &controller.Subscription{
+		ReconcileFunc: NewAuthorinoCrReconciler(client).Reconcile,
+		Events: []controller.ResourceEventMatcher{
+			{Kind: ptr.To(kuadrantv1beta1.KuadrantKind), EventType: ptr.To(controller.CreateEvent)},
```

Comment:
Why Kuadrant CREATE only? What about when it's updated (with changes affecting the Authorino CR) or deleted?

Reply:
The lack of delete is the easy one. Because of the owner reference when the kuadrant CR is fully remove from the cluster kubernetes will delete the Authorino CR. Delete is an event that we don't beed to handle.

The kuadrant CR update events also don't matter. There is no spec setable in the kuadrant CR that will affect the authorino CR. That feature was removed from the kuadrant spec for V1 as it is going to be redone to suit what is outlined in https://github.com/Kuadrant/architecture/pull/97 , so update events also don't matter.

Only the create events matter.